### DISCUSSION TITLE
Migration / mirroring of the source code from GitHub to a free alternative source code delivery platform (e.g. the alternative based on Gitea and operated by an German NGO on German / European server infrastructure: Codeberg)

### DISCUSSION THREAD

**Discussion by linux-lukas** [SchoolUtils/WebUntis#78] [General]
Given GitHub's obvious dependence on Microsoft (I mean: Microsoft bought GitHub in 2018), the question would be whether it would not make sense to rely on an alternative platform for source code provision. 

The alternative called Codeberg, which is based on Gitea and operated by Codeberg e. V., would make sense.

**Comment by luwol03** [comment]
What are the downsides of using github? It's free and perfect to grow in publicity imho, because everyone can find your project.

**Comment by TheNoim** [comment]
I can see creating mirrors on other hosts, but the source repo will stay on Github. Even though I see the point, Github is by far more accessible and discoverable. This is important, especially for smaller projects like this. 

In addition, Github fits the situation more, because this project probably attracts newcomer programmers who have not much experience. And I think Github is more beginner friendly.
